atomic_reactor/source.py

```diff
@@ -172,6 +173,38 @@ def get(self):
         return self.source_path
 
 
+class DummySource(Source):
+    """Dummy source that just provides defaults in cases where we don't expect
+     operations with real data
+    """
+    def __init__(
+        self, provider, uri, dockerfile_path=None,
+        provider_params=None, tmpdir=None
+    ):
+        # intentionally not calling `super()`
+        self._config = None
+        self.provider = provider
+        self.uri = uri
+        self.dockerfile_path = dockerfile_path
+        self.provider_params = provider_params or {}
+        # TODO: do we want to delete tmpdir when destroying the object?
+        self.tmpdir = tmpdir or tempfile.mkdtemp()
+        logger.debug("workdir is %r", self.tmpdir)
+        self.source_path = tempfile.mkdtemp()
```

Comment:
Could be self.tmpdir?

Reply:
I didn't want to mess `work dir` with `source dir` to prevent writing temp data into source directory (not an issue now but who knows in future)

Reply:
I can use:
```
self.source_path = tempfile.mkdtemp(dir=self.tmpdir)
```
This seems better